### ISSUE TITLE
Jump To command not keyboard accessible

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7070]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.448
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- open an R script containing multiple functions in the source editor
- invoke the "Jump To..." command (Code / Jump To)
- try to navigate the list of functions with the keyboard arrows

### Describe the problem in detail
- keyboard doesn't change the selection; this is a regression from RStudio 1.3 and earlier

### Describe the behavior you expected
- should be able to operate the popup with keyboard

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
This is a regression caused by #6576 starting in build 1.4.134.

**Comment by kfeinauer** [comment]
Verified fixed on both RStudio 1.4.944 and RSP 1.4.940-1
